src/models/event.ts

```diff
@@ -913,9 +903,10 @@ export class MatrixEvent extends TypedEventEmitter<MatrixEventEmittedEvents, Mat
                 content: {
                     msgtype: "m.bad.encrypted",
                     body: "** Unable to decrypt: " + reason + " **",
-                    reason,
                 },
             },
+            isEncryptedDisabledForUnverifiedDevices:
+                reason === "DecryptionError: The sender has disabled encrypting to unverified devices.",
```

Comment:
It'd be great here if we were using the source of truth which iirc is WITHHELD_MESSAGES

as for the previous approach vs this one, either is fine, but `MatrixEvent::isEncryptedDisabledForUnverifiedDevices` is mutable so should really be behind a getter

Reply:
When the event is decrypted, `crypto.decryptEvent(this)` is called and the string `DecryptionError: The sender has disabled encrypting to unverified devices."` is the content of the error raised. How can I have access to the WITHHELD_MESSAGES ?

Reply:
So the reason field is a stringified `algorithms.DecryptionError`

It's still not ideal but I'd suggest `reason === "DecryptionError: " + WITHHELD_MESSAGES["m.unverified"]` or similar, rather than duplicating the user-facing string.

Reply:
Oh right thanks

Reply:
I used `WITHHELD_MESSAGES `, it should be cleaner.